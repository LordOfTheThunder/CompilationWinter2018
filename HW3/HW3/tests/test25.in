int func(int x, int y) {
	x = 3;
}
