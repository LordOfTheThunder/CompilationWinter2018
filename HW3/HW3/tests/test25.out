---end scope---
y INT -2
x INT -1
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
func (INT,INT)->INT 3
